### PR TITLE
Simplify configuration parameters

### PR DESCRIPTION
**Pull request by alexpan0610**
This PR can not pass CI test until https://github.com/elastos/Elastos.ELA.SideChain/pull/242 merged.
This PR can not pass CI test until https://github.com/elastos/Elastos.ELA.SPV/pull/90 merged.

### CODE REVIEW

**Comment by alexpan0610** [comment]
https://github.com/elastos/Elastos.ELA.TestManagement/issues/24
https://github.com/elastos/Elastos.ELA.TestManagement/issues/40

**Comment by alexpan0610** [comment]
This PR can not pass CI test until elastos/Elastos.ELA.SPV#90 merged.

File: config.go [hunk 1]
```diff
@@ -1,215 +1,184 @@
...
+	DisableDNS         bool
+	PermanentPeers     []string
+	SPVDNSSeeds        []string
+	SPVDisableDNS      bool
+	SPVPermanentPeers  []string
+	EnableREST         bool
+	RESTPort           uint16
+	EnableWS           bool
+	WSPort             uint16
+	DisableRPC         bool
```
Comment (bocheng0000) [inline config.go]: DisableRPC should be consistent with EnableREST

**Comment by bocheng0000** [comment]
Please add documentation for config.json, especially the newly added parameters.

File: config.go (hunk 1 above)
Comment (alexpan0610) [reply config.go]: The RPC service is enabled by default, and the REST and WS is disabled by default, the DisableRPC and EnableREST parameter is for different usage, it's not about consistent.

Comment (bocheng0000) [reply config.go]: Configuration items should be consistent.
The default startup can be achieved by setting the default value of EnableRPC to True.

Comment (alexpan0610) [reply config.go]: The DisableRPC is keeping consistent with the main chain configuration, of course you can change it to EnableRPC, it's depends on your choice.

Comment (bocheng0000) [reply config.go]: There is no EnableRPC in mainchain[release_v0.3.1].
If EnableRPC is added in the next version, it should be removed or changed to EnableRPC.

**Comment by alexpan0610** [comment]
Documentation added in `docs/config.json.md`.

**Pull request merged**